lib/matplotlib/backends/backend_svg.py

```diff
@@ -481,14 +490,6 @@ def _make_id(self, type, content):
     def _make_flip_transform(self, transform):
         return transform + Affine2D().scale(1, -1).translate(0, self.height)
 
-    def _get_font(self, prop):
```

Comment:
This is pretty unrelated; it should be a separate commit at least.

Reply:
I can see why it looks like that. ;-) (Originally, the purpose of the PR was to increase test coverage.)

I've moved it to a separate PR.